kivy/lib/osc/OSC.py

```diff
@@ -108,9 +139,21 @@ def readString(data):
 
 
 def readBlob(data):
-    length   = struct.unpack(">i", data[0:4])[0]
-    nextData = int(math.ceil((length) / 4.0) * 4) + 4
-    return (data[4:length+4], data[nextData:])
+    if(len(data)<4):
```

Comment:
Was just applying the style found there. Changing it to try/except makes more sense.